src/Stratis.Features.FederatedPeg/SourceChain/Deposit.cs

```diff
@@ -12,6 +13,7 @@ public Deposit(uint256 id, DepositRetrievalType retrievalType, Money amount, str
             this.RetrievalType = retrievalType;
             this.Amount = amount;
             this.TargetAddress = targetAddress;
+            //this.TargetChain = targetChain;
```

Comment:
Is this going to be introduced later?

Reply:
Yes. Adding it in the constructor breaks a lot of tests, I was about to work on it but had to switch to other task, so disabled it in this PR.